### DISCUSSION TITLE
October 2024 Survey Results - Question 11

### DISCUSSION THREAD

**Discussion by boc-the-git** [mealie-recipes/mealie#4649] [General]
Placeholder discussion for people to share their thoughts and insights about the survey results.

**Comment by ChipSkylark37** [comment]
I wasn't able to participate in the survey, but leaving some comments here on the question regarding the Timeline:

1. Default sort to Descending Order. 

I dont need to see the first recipe I ever created by default. It should at least be an option

2. Ability to edit the 'I Made This' 

Sometimes I want to edit the image (rotating, separate issue that I have an open bug / discussion on)  Maybe I accidently selected the wrong date that I made something, or want to add a picture later. I see you can edit / add a comment if you forgot to do that, which is nice, but sometimes the default image doesnt come up properly and I want to change / replace it.

3. Ability to Filter More

Sometimes I want to just filter to what I've made vs the new recipes I've added. A filter for 'Recipe Created' vs 'I Made This' would make this much more useful to see just dishes I've made.

Additionally, being able to filter by Home / Group / User that made something. I share an instance with a friend and if they made something, I'd like to be able to (by default) filter it to just the logged in user (myself in this instance) BUT I would like to be able to see what they made if I choose to include it in a filter. 

4. Scaling / Compact Mode

Overall, there's a lot of "white space' on the screen. I'd like a much more compact view. Maybe options to de-select the picture of the recipe and just have the name? or being able to fit more onto the screen. 

5. Summary (By Date?)

I'd like to be able to see things like "XX Dishes made in the last 2 weeks" or "YY new recipes added since (user selected date". Sort of just like overall statistics.

**Reply by Iteria** [reply]
I definitely agree with the default to descending order. The timeline is pretty unusable as you continue use without this. Especially if you have a group that's pretty active.

**Comment by rabidfurball** [comment]
I don't really see the point of having a whole separate timeline page, and would prefer to just keep notes and photos on each individual recipe page.  But I would like to expand the comment ability on each recipe to include images (and wish all image uploads supported pasting images into the page rather than only uploading from a file picker).  

Basically I think hitting "I made this" should just post a special comment on the recipe that logs the date, maybe location, allows for some notes about the prep, an image, and possibly would allow for rating that specific preparation (in case you made any ingredient changes that time or something).  A filter for the comments to filter "i made this" comments from general comments, and to sort by comment ratings, and a simple image gallery to quickly show all recipe photos together would also be nice features. I would also want the whole comments section to be collapsed by default, and only load if clicked on, for better performance.

**Reply by michael-genson** [reply]
We've gotten a lot of similar feedback regarding the timeline, so I think this makes sense. We should be able to convert existing events into comments and add some extra metadata on the comment along with image support

**Reply by Iteria** [reply]
I want to jump in and add: who is saying that the timeline is not useful? If you have a single household then no it wouldn't but. For me who is collaborating with a group of my friends the timeline has value in seeing what people are adding and also what they are making. I like to see their pictures. Honestly, I'd like to comment on what they added when they say they cooked something. Shunting everything into comments would significantly reduce my groups ability to keep up with each other. If Mealie is not meant to be social, then sure this makes sense, but why have comments at all if there is not meant to be on app cross collaboration and discussion?
